test/AutoDiff/differentiable_function_silgen.swift

```diff
@@ -1,6 +1,5 @@
 // RUN: %target-swift-frontend -dump-ast %s | %FileCheck %s -check-prefix=CHECK-AST
 // RUN: %target-swift-frontend -emit-silgen %s | %FileCheck %s -check-prefix=CHECK-SILGEN
-// RUN: %target-swift-frontend -emit-sil %s | %FileCheck %s -check-prefix=CHECK-SIL
```

Comment:
This RUN line was removed because `differentiable_function_silgen.swift` is supposed to be a SILGen test, not a SIL canonicalization test.